<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpPage</class>
 <widget class="QMainWindow" name="SignUpPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1920</width>
      <height>1080</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>LogInPic.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="LogInLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>911</width>
      <height>591</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(90, 24, 154);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>230</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(223, 126, 255);</string>
    </property>
    <property name="text">
     <string>Sign Up </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>531</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(223, 126, 255);</string>
    </property>
    <property name="text">
     <string>Choose your Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>380</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(223, 126, 255);</string>
    </property>
    <property name="text">
     <string>Choose your password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewUsernameInput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>350</y>
      <width>821</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 119, 231);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="NewPasswordInput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>821</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 119, 231);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SignUpBtn">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>720</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;Arial Rounded MT Bold&quot;;
	font: 75 10pt &quot;Georgia&quot;;
    background-color: rgb(171, 119, 231);
    color: white;
    border: none;
    border-radius: 40px;
	
}



QPushButton:pressed {
    background-color: rgb(131, 79, 191);
}


</string>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>620</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(223, 126, 255);</string>
    </property>
    <property name="text">
     <string>Do you agree to the terms and policies?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AgreementCheckBox">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>620</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(223, 126, 255);
font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorLabel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>680</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>840</y>
      <width>801</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(223, 126, 255);</string>
    </property>
    <property name="text">
     <string>Already have an account?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LogInbtn">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>840</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	font: 20pt &quot;Arial Rounded MT Bold&quot;;
	font: 75 10pt &quot;Georgia&quot;;
    background-color: rgb(171, 119, 231);
    color: white;
    border: none;
    border-radius: 10px;
	
}



QPushButton:pressed {
    background-color: rgb(131, 79, 191);
}


</string>
    </property>
    <property name="text">
     <string>Log In</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(223, 126, 255);</string>
    </property>
    <property name="text">
     <string>Retype the password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RetypePasswordInput">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>530</y>
      <width>821</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(171, 119, 231);
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SignUpErrorLabel">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>570</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>LogInLabel</zorder>
   <zorder>NewUsernameInput</zorder>
   <zorder>NewPasswordInput</zorder>
   <zorder>label_3</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>SignUpBtn</zorder>
   <zorder>label_7</zorder>
   <zorder>AgreementCheckBox</zorder>
   <zorder>ErrorLabel</zorder>
   <zorder>label_8</zorder>
   <zorder>LogInbtn</zorder>
   <zorder>label_9</zorder>
   <zorder>RetypePasswordInput</zorder>
   <zorder>SignUpErrorLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
